-- -*- lua -*-
-- Module file created by spack (https://github.com/spack/spack) on 2019-09-16 14:34:15.500706
--
-- r-ape@5.0%gcc@5.5.0 arch=linux-centos7-x86_64 /rn7mysd
--

whatis([[Name : r-ape]])
whatis([[Version : 5.0]])
whatis([[Short description : Functions for reading, writing, plotting, and manipulating phylogenetic trees, analyses of comparative data in a phylogenetic framework, ancestral character analyses, analyses of diversification and macroevolution, computing distances from DNA sequences, reading and writing nucleotide sequences as well as importing from BioConductor, and several tools such as Mantel's test, generalized skyline plots, graphical exploration of phylogenetic data (alex, trex, kronoviz), estimation of absolute evolutionary rates and clock-like trees using mean path lengths and penalized likelihood, dating trees with non-contemporaneous sequences, translating DNA into AA sequences, and assessing sequence alignments. Phylogeny estimation can be done with the NJ, BIONJ, ME, MVR, SDM, and triangle methods, and several methods handling incomplete distance matrices (NJ*, BIONJ*, MVR*, and the corresponding triangle method). Some functions call external applications (PhyML, Clustal, T-Coffee, Muscle) whose results are returned into R.]])

help([[Functions for reading, writing, plotting, and manipulating phylogenetic
trees, analyses of comparative data in a phylogenetic framework,
ancestral character analyses, analyses of diversification and
macroevolution, computing distances from DNA sequences, reading and
writing nucleotide sequences as well as importing from BioConductor, and
several tools such as Mantel's test, generalized skyline plots,
graphical exploration of phylogenetic data (alex, trex, kronoviz),
estimation of absolute evolutionary rates and clock-like trees using
mean path lengths and penalized likelihood, dating trees with non-
contemporaneous sequences, translating DNA into AA sequences, and
assessing sequence alignments. Phylogeny estimation can be done with the
NJ, BIONJ, ME, MVR, SDM, and triangle methods, and several methods
handling incomplete distance matrices (NJ*, BIONJ*, MVR*, and the
corresponding triangle method). Some functions call external
applications (PhyML, Clustal, T-Coffee, Muscle) whose results are
returned into R.]])



prepend_path("LD_LIBRARY_PATH", "/u/local/spack/0.12/opt/spack/linux-centos7-x86_64/gcc-5.5.0/r-ape-5.0-rn7mysdgetpvh6yf5bnwd63vud5zpbtn/lib", ":")
prepend_path("CMAKE_PREFIX_PATH", "/u/local/spack/0.12/opt/spack/linux-centos7-x86_64/gcc-5.5.0/r-ape-5.0-rn7mysdgetpvh6yf5bnwd63vud5zpbtn/", ":")
prepend_path("R_LIBS", "/u/local/spack/0.12/opt/spack/linux-centos7-x86_64/gcc-5.5.0/r-ape-5.0-rn7mysdgetpvh6yf5bnwd63vud5zpbtn/rlib/R/library", ":")
prepend_path("XDG_DATA_DIRS", "/u/local/spack/0.12/opt/spack/linux-centos7-x86_64/gcc-5.5.0/pango-1.41.0-43ieu7cr4heyqb4ltevhxgyf2uphjnd5/share", ":")
setenv("R_APE_ROOT", "/u/local/spack/0.12/opt/spack/linux-centos7-x86_64/gcc-5.5.0/r-ape-5.0-rn7mysdgetpvh6yf5bnwd63vud5zpbtn")

